Makefile.buildx.mk

```diff
@@ -1,18 +1,3 @@
-
-# Copyright 2018-2020 The OpenEBS Authors. All rights reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
```

Comment:
why are the license removed. IMHO  the `2018-2020`  should be updated to `2018-2024`

Reply:
Because we are not requied to put them on top of every file, so it was decided in the maintainers meeting that whenever someone changes a file they are supposed to remove this from the file header.